build.sbt

```diff
@@ -120,12 +120,24 @@ lazy val spark = Project("spark", file("spark")).
   dependsOn(util, raster).
   settings(commonSettings: _*)
 
+lazy val s3 = Project("s3", file("backends/s3")).
+  dependsOn(spark).
```

Comment:
I don't understand all of the implications either, but I can certainly make that change.  Will do the same for the accumulo backend.